3party/opening_hours/opening_hours_tests/opening_hours_tests.cpp

```diff
@@ -1262,6 +1334,57 @@ BOOST_AUTO_TEST_CASE(OpeningHours_TestIsActive)
     BOOST_CHECK(GetTimeTuple("2010-4-20-0 18:15", fmt, time));
     BOOST_CHECK(IsActive(rules[0], time));
   }
+  {
+    TRuleSequences rules;
+    BOOST_CHECK(Parse("Mo 09:00-06:00", rules));
+
+    std::tm time{};
+    auto const fmt = "%Y-%m-%d %H:%M";
```

Comment:
This format string is used multiple times in this test, maybe it's worth to move it up an reuse?